server/src/server.ts

```diff
@@ -33,8 +34,8 @@ const bitBakeDocScanner = new BitBakeDocScanner()
 const bitBakeProjectScanner: BitBakeProjectScanner = new BitBakeProjectScanner(connection)
 const contextHandler: ContextHandler = new ContextHandler(bitBakeProjectScanner)
 
-connection.onInitialize(async (params): Promise<InitializeResult> => {
-  const workspaceRoot = params.rootPath ?? ''
+connection.onInitialize(async (params: InitializeParams): Promise<InitializeResult> => {
+  const workspaceRoot = params.workspaceFolders?.[0]?.uri ?? ''

```

Comment:
I think it is possible that users have multiple folders opened in one workspace and all of them are bitbake projects.